res/controllers/AKAI-MPKmini2-scripts.js

```diff
@@ -0,0 +1,139 @@
+
+var MPKmini = {};
+
+MPKmini.init = function() {
+    this.currentUnitNum = 1;
+    this.currentEffectNum = 1;
+    this.currentFlash = true;
+    this.timerId = 0;
+
+    MPKmini.updateLed();
+    this.timerId = engine.beginTimer(200, "MPKmini.flashLed()");
+};
+
+MPKmini.shutdown = function() {
+    if (this.timerId !== 0)
+        engine.stopTimer(this.timerId);
+    else
+        print("Shutdown: no timer to stop.");
+};
+
+MPKmini.getCurrentGroup = function(withEffect, withUnit) {
+    var grp = "";
+    if (withEffect) {
+        grp =  "[EffectRack1_EffectUnit" + MPKmini.currentUnitNum + "_Effect" + MPKmini.currentEffectNum + "]";
+    } else if (withUnit) {
+        grp =  "[EffectRack1_EffectUnit" + MPKmini.currentUnitNum + "]";
+    } else {
+        grp =  "[EffectRack1]";
+    }
+    print("get group: "+grp);
+    return grp;
```

Comment:
No need to save 2 chars here.

```suggestion
    var group = "";
    if (withEffect) {
        group =  "[EffectRack1_EffectUnit" + MPKmini.currentUnitNum + "_Effect" + MPKmini.currentEffectNum + "]";
    } else if (withUnit) {
        group =  "[EffectRack1_EffectUnit" + MPKmini.currentUnitNum + "]";
    } else {
        group =  "[EffectRack1]";
    }
    print("get group: "+group);
    return group;
```